apps/fabric-website/src/pages/HomePage/HomePage.base.tsx

```diff
@@ -211,15 +210,15 @@ export class HomePageBase extends React.Component<IHomePageProps, IHomePageState
                     beakWidth: 8,
                     isBeakVisible: true,
                     shouldFocusOnMount: true,
-                    items: fabricVersionOptions,
+                    items: versionOptions,
                     directionalHint: DirectionalHint.bottomCenter,
-                    onItemClick: this._onVersionMenuClick,
+                    onItemClick: onVersionMenuClick,
                     styles: {
                       root: { minWidth: 100 },
                     },
                   }}
                 >
-                  Fluent UI React {reactPackageData.version}
+                  Fluent UI React {currentVersionNumber}
```

Comment:
Should this say `Fabric` if it's actually a fabric release?

Reply:
Yes, but this version of the code is only used in v7 where we want to call it fluent.